src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -335,9 +336,10 @@ public RestClientMethod GetRestClientMethod(Operation operation)
                 foreach (var restClientMethod in restClient.Methods)
                 {
                     // skip all internal methods
-                    if (restClientMethod.Accessibility != MethodSignatureModifiers.Public)
-                        continue;
-                    restClientMethods.Add(restClientMethod.Operation, restClientMethod);
+                    if (restClientMethod.Accessibility == MethodSignatureModifiers.Public && restClientMethod.Operation is CodeModelOperation operation)
```

Comment:
This doesn't seem to match what we discussed earlier around the swagger path for mgmt plane shouldn't have any changes since it will go Yaml -> CodeModelClasses -> MPG.  The two new paths we discussed were CADL -> JSON -> InputClasses -> NewDPG and Yaml -> CodeModelClasses -> InputClasses -> NewDPG.

We should re-evaluate our new class mapping here and validate that what we intended is what ended up happening

Reply:
This is temporary, since InputClasses don't support models.